### ISSUE TITLE
kernel R not work with Spark 2.4.0 

### ISSUE THREAD

**Issue by zgfh** [jupyter/docker-stacks#766]
**What docker image you are using?**
jupyter/all-spark-notebook

**What complete docker command do you run to launch the container (omitting sensitive values)?**

`docker run -it --rm -p 8888:8888 jupyter/all-spark-notebook`

**What steps do you take once the container is running to reproduce the issue?**

1. Visit http://localhost:8888
2. Start an R noteobok
3. run code 
```
library(sparklyr)
sc <- spark_connect(master = "local")
```
**What do you expect to happen?**
 sc of spark

**What actually happens?**

![image](https://user-images.githubusercontent.com/11729624/48993632-f1828600-f179-11e8-8dcc-92ec95ff0a15.png)

sparklyr release note: https://github.com/rstudio/sparklyr/blob/master/NEWS.md
should we update version of sparklyr to 0.9.2?

**Comment by parente** [comment]
Looks like Spark 2.4.0 support is still in the works: https://github.com/rstudio/sparklyr/issues/1749

**Comment by romainx** [comment]
Hello,
Spark `2.4` is supported in Sparklyr `0.9.3`.

The information is given in the [release note](https://github.com/rstudio/sparklyr/blob/master/NEWS.md#sparklyr-093).
And also in the package.
```r
> packageVersion("sparklyr")
[1] ‘0.9.3’
> spark_available_versions() %>% tail(2)
   spark
14 2.3.2
15 2.4.0
```

**Comment by parente** [comment]
Thanks @romainx. Looks like it dropped 8 days ago. If someone would like to submit a PR with the package version bump we can get it merged.

**Comment by zgfh** [comment]
@parente https://github.com/jupyter/docker-stacks/pull/773,
but I don't know if it will broken something

**Comment by parente** [comment]
Fixed by #782.
